src_override/constants.F90

```diff
@@ -0,0 +1,273 @@
+real, public, parameter :: RADCON_MKS  = (GRAV/CP_AIR)*SECONDS_PER_DAY !< Factor used to convert flux divergence to
+                                                                       !! heating rate in degrees per day [deg sec/(m day)]
+real, public, parameter :: O2MIXRAT    = 2.0953E-01_r8_kind !< Mixing ratio of molecular oxygen in air [dimensionless]
+real, public, parameter :: RHOAIR      = 1.292269_r8_kind   !< Reference atmospheric density [kg/m^3]
+real, public, parameter :: VONKARM     = 0.40_r8_kind       !< Von Karman constant [dimensionless]
+real, public, parameter :: C2DBARS     = 1.e-4_r8_kind      !< Converts rho*g*z (in mks) to dbars: 1dbar = 10^4 (kg/m^3)(m/s^2)m [dbars]
+real, public, parameter :: KELVIN      = 273.15_r8_kind     !< Degrees Kelvin at zero Celsius [K]
+
+#else
+
+
+use shr_kind_mod,   only : R8 => shr_kind_r8
+use shr_const_mod,  only : SHR_CONST_PI, &
+     SHR_CONST_CDAY, &
+     SHR_CONST_OMEGA, &
+     SHR_CONST_REARTH, &
+     SHR_CONST_G, &
+     SHR_CONST_STEBOL,&             !   = 5.67e-8_R8      ! Stefan-Boltzmann constant ~ W/m^2/K^4
+     SHR_CONST_AVOGAD,&             !   = 6.02214e26_R8   ! Avogadro's number ~ molecules/kmole
+     SHR_CONST_MWDAIR,&             !   = 28.966_R8       ! molecular weight dry air ~ kg/kmole
+     SHR_CONST_MWWV,&               !   = 18.016_R8       ! molecular weight water vapor
+     SHR_CONST_RDAIR,&              ! Dry air gas constant     ~ J/K/kg
+     SHR_CONST_RWV,&                ! Water vapor gas constant ~ J/K/kg
+     SHR_CONST_KARMAN,&             !   = 0.4_R8          ! Von Karman constant
+     SHR_CONST_PSTD,&               !   = 101325.0_R8     ! standard pressure ~ pascals
+     SHR_CONST_TKFRZ,&              !   = 273.15_R8       ! freezing T of fresh water          ~ K
+     SHR_CONST_RHODAIR,&            ! density of dry air at STP  ~ kg/m^3
+     SHR_CONST_RHOFW,&              !   = 1.000e3_R8      ! density of fresh water     ~ kg/m^3
+     SHR_CONST_RHOSW,&              !   = 1.026e3_R8      ! density of sea water       ~ kg/m^3
+     SHR_CONST_CPDAIR,&             !   = 1.00464e3_R8    ! specific heat of dry air   ~ J/kg/K
+     SHR_CONST_CPWV,&               !   = 1.810e3_R8      ! specific heat of water vap ~ J/kg/K
+     SHR_CONST_CPSW,&
+     SHR_CONST_LATICE,&             !   = 3.337e5_R8      ! latent heat of fusion      ~ J/kg
+     SHR_CONST_LATVAP               !   = 2.501e6_R8      ! latent heat of evaporation ~ J/kg
+
+implicit none
+private
+
+! Include variable "version" to be written to log file.
+#include<file_version.h>
+!-----------------------------------------------------------------------
+! version is public so that write_version_number can be called for constants_mod
+! by fms_init
+public :: version
+
+real :: realnumber !< dummy variable to use in HUGE initializations
+
+!-----------------------------------------------------------------------
```

Comment:
You are creating new paramater variables here when all you need to do is rename the existing ones.
Please changes this, for example:
use shr_const_mod, only RADIUS=>SHR_CONST_REARTH